### PR TITLE
Completed typescript-arrays exercise.

### PR DESCRIPTION
**Pull request by Katelyn-Ngo**
Screen Shot 2024-05-14 at 1 56 53 PM

### CODE REVIEW

File: typescript-arrays/main.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const thirdColor: string = colors[2];
+console.log('value of colors[0]:', firstColor);
+console.log('value of colors[1]:', secondColor);
+console.log('value of colors[2]:', thirdColor);
+
+const AmericaColors: string = `America is ${firstColor} ${secondColor} ${thirdColor}`;
+console.log(AmericaColors);
+
+colors[2] = 'green';
+const MexicoColors: string = `Mexico is ${firstColor} ${secondColor} ${thirdColor}`;
```
Comment (brandon-moy) [inline typescript-arrays/main.ts]: You did this correctly on line 12 where you updated colors[2] to be green, but if you look at your console it shows "Mexico is red white blue" and Mexico should be red white green.

File: typescript-arrays/main.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+console.log(MexicoColors);
+
+console.log('value of colors:', colors);
+console.log('typeof colors:', typeof colors);
+
+const students: string[] = ['charlie', 'billy', 'ken', 'ben'];
+const numberOfStudents: number = students.length;
+console.log('There are four students:', numberOfStudents);
+const lastIndex: number = numberOfStudents - 1;
+console.log('the last student in array is', lastIndex);
```
Comment (brandon-moy) [inline typescript-arrays/main.ts]: You don't need this line 23 because we don't need the lastIndex to be logged. It also makes it look like "the last student in the array is 3" whereas your console log on line 26 is the correct value of "the last student in the array is ben"

tl;dr - you can remove line 23

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Couple of changes requested but otherwise looks good 👍

**Review by brandon-moy — APPROVED** [review]
Looks good! I would say that if you are going to use the `colors[2]` for the green, you should make your template literal consistent so using `colors[0]` and `colors[1]` but approved!
